### PR TITLE
New resource properties

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves https://github.com/Twingate/terraform-provider-twingate/issues/251

## Changes
- added optional resource properties: is_visible and is_browser_shortcut_enabled

### CODE REVIEW

File: twingate/internal/provider/resource/resource.go [hunk 1]
```diff
@@ -102,6 +102,16 @@ func Resource() *schema.Resource { //nolint:funlen
 					},
 				},
 			},
+			"is_visible": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Indicates if the Resource is active.",
```
Comment (vmanilo) [inline twingate/internal/provider/resource/resource.go:108]: @alexb-twingate please check doc string

File: twingate/internal/provider/resource/resource.go [hunk 2]
```diff
@@ -102,6 +102,16 @@ func Resource() *schema.Resource { //nolint:funlen
...
+			},
+			"is_browser_shortcut_enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Indicates whether this Resource will display a browser shortcut in the client.",
```
Comment (vmanilo) [inline twingate/internal/provider/resource/resource.go:113]: @alexmensch please check doc string

**Review by ekampf — APPROVED** [review]

**Review by ekampf — APPROVED** [review]
Code looks good. Description for `is_visible` is wrong I'll wait for @alexmensch  to give a fix before merging

File: docs/resources/resource.md [hunk 3]
```diff
@@ -56,6 +56,8 @@ resource "twingate_resource" "resource" {
 ### Optional
 
 - `group_ids` (Set of String, Deprecated) List of Group IDs that have permission to access the Resource, cannot be generated by Terraform and must be retrieved from the Twingate Admin Console or API
+- `is_browser_shortcut_enabled` (Boolean) Indicates whether this Resource will display a browser shortcut in the client.
```
Comment (alexmensch) [inline docs/resources/resource.md:59]: Description here should be: `Controls whether an "Open in Browser" shortcut will be shown for this Resource in the Twingate Client.`

File: docs/resources/resource.md [hunk 4]
```diff
@@ -56,6 +56,8 @@ resource "twingate_resource" "resource" {
...
+- `is_visible` (Boolean) Indicates if the Resource is active.
```
Comment (alexmensch) [inline docs/resources/resource.md:60]: Description here: `Controls whether this Resource will be visible in the main Resource list in the Twingate Client.`

**Review by alexmensch — CHANGES_REQUESTED** [review]

**Review by alexmensch — APPROVED** [review]

**Pull request merged**